src/states/elder/mod.rs

```diff
@@ -542,33 +550,16 @@ impl Elder {
     // After parsec reset, resend any unaccumulated `NeighbourInfo` messages to everyone that needs
     // them but possibly did not receive them already. That is, resend to all newly promoted elders
     // in our section and in case of split also to elders of the sibling section.
-    fn resend_neighbour_infos(&mut self, old_pfx: Prefix<XorName>) {
-        let sibling = if self.our_prefix().is_extension_of(&old_pfx) {
-            Some(self.our_prefix().sibling())
-        } else {
-            None
-        };
-
-        let sibling_is_neighbour = |src_prefix: &Prefix<_>| {
-            sibling
-                .map(|sibling| sibling.is_neighbour(src_prefix))
-                .unwrap_or(false)
-        };
-
-        for (_, msg) in mem::replace(&mut self.pending_neighbour_info_msgs, HashMap::new()) {
-            let src_prefix = msg.elders_info.prefix();
-
-            if self.our_prefix().is_neighbour(src_prefix) || sibling_is_neighbour(src_prefix) {
-                let msg = msg.into();
-                match self.send_signed_message(&msg) {
-                    Ok(()) => trace!("{} - Resend {:?}", self, msg.routing_message()),
-                    Err(error) => debug!(
-                        "{} - Failed to resend {:?}: {:?}",
-                        self,
-                        msg.routing_message(),
-                        error
-                    ),
-                }
+    fn resend_neighbour_infos(&mut self, _old_pfx: Prefix<XorName>) {
```

Comment:
This is no longer used just for `NeighbourInfo` (or at least it doesn't have to be), so this method should be renamed. What about `resend_pending_voted_messages` ? Also the comment should be updated.